livekit-ffi/protocol/room.proto

```diff
@@ -210,6 +210,7 @@ message RoomEvent {
     Reconnecting reconnecting = 19;
     Reconnected reconnected = 20;
     E2eeStateChanged e2ee_state_changed = 21;
+    RoomEOS eos = 22;
```

Comment:
what does it mean to be eos?

Reply:
that the stream of events is finished. so we can stop listening

Reply:
happy to hear better names